packages/generator/src/parser/parser-types-v4.ts

```diff
@@ -54,3 +57,13 @@ export interface EdmxNavigationPropertyBinding {
   Path: string;
   Target: string;
 }
+
+export interface EdmxFunction extends EdmxNamed {
+  ReturnType: { Type: string };
+  Parameter: EdmxParameter[];
+  IsBound: boolean;
```

Comment:
Is there any tests/use cases about this?

Reply:
Isbound? No it is not used anywhere yet.